testsupport/space.go

```diff
@@ -231,7 +231,7 @@ func verifyResourcesProvisionedForSpace(t *testing.T, hostAwait *wait.HostAwaiti
 }
 
 func CreateMurWithAdminSpaceBindingForSpace(t *testing.T, awaitilities wait.Awaitilities, space *toolchainv1alpha1.Space, cleanup bool) (*toolchainv1alpha1.UserSignup, *toolchainv1alpha1.MasterUserRecord, *toolchainv1alpha1.SpaceBinding) {
-	username := "for-space-" + space.Name
+	username := space.Name
```

Comment:
with this function, we wanted to provision a MUR to have the SpaceBinding set so the Space doesn't get deleted. this means that the User would have access to two spaces:
1. the one that was created as a standalone Space
2. the one that is provisioned for the user by default

That was the reason for introducing the prefix `for-space-` - to differentiate the names of the Spaces. 
However, since the name would be the same, then the UserSignup controller woudn't provision a separate Space CR, but would reuse the already existing one. 

That being said - could we change the prefix to:
```go
username := "sp-" + space.Name
```

Reply:
But unless I'm missing something, the function triggers a signups request with `NoSpace()` few lines above, it doesn't seem to provision the second space.But it's  also Monday , so probably I just need more coffe ☕ 😄

Reply:
ah, OK. I didn't see the line. Yeah, you are right then :+1: